consts.go

```diff
@@ -22,9 +22,17 @@ const (
 	recoveryStateWipe
```

Comment:
Should we make those human readable as well? Like 'c\n', 'u\n', 'w\n'

Reply:
perhaps. Might be worth punting to a follow up pr.